src/test/java/com/softserveinc/ita/SearchTest.java

```diff
@@ -50,14 +51,17 @@ public void verifyRecentlyViewedProducts() {
 
         var lastViewedItemFromMainPage = homePage
                 .clickOnMainPageLogo()
-                .homePageLastViewedProductTitle(expectedItem);
+                .selectFirstItemFromLastProduct();
 
-        assertThat(lastViewedItemFromMainPage)
+        assertThat(lastViewedItemFromMainPage
+                .stream()
+                .map(SelenideElement::text)
+                .filter(text -> text.contains(expectedItem)))
```

Comment:
rather you will not need filtering here, just insert into assert the list and contains should find